web/src/index.js

```diff
@@ -23,21 +23,109 @@ const authMiddleware = new ApolloLink((operation, forward) => {
   operation.setContext({
     headers: {
       authorization: Cookie.get('token') || null,
-    }
+    },
   });
   return forward(operation);
-})
+});
+
+// TODO(YoonYeoHwan): Remove this before merge
```

Comment:
maybe remove this now??

Reply:
Now, back-end is refactoring. So I can't test my codes without these codes.